app/views/stripe-setup/responsible-person/index.njk

```diff
@@ -0,0 +1,156 @@
+{% extends "../../layout.njk" %}
+{% from "components/input/macro.njk" import govukInput %}
+{% from "components/button/macro.njk" import govukButton %}
+{% from "components/date-input/macro.njk" import govukDateInput %}
+{% from "components/error-summary/macro.njk" import govukErrorSummary %}
```

Comment:
Doesn't look like you need `govukErrorSummary` here

Reply:
Not yet but we will soon 😀